app/default_overrides.go

```diff
@@ -294,5 +294,8 @@ func DefaultAppConfig() *serverconfig.Config {
 	cfg.StateSync.SnapshotInterval = 1500
 	cfg.StateSync.SnapshotKeepRecent = 2
 	cfg.MinGasPrices = fmt.Sprintf("%v%s", appconsts.DefaultMinGasPrice, BondDenom)
+
+	const mebibyte = 1048576
```

Comment:
a super tiny nit for a follow up issue would be to extract the `mebibyte` constant and export it somewhere. We have a lot of redefinitions of it, so it would be good to use the same one to avoid any mistakes